### PR TITLE
add foot IK targets as actions, map Kinect feet to foot targets

### PR DESCRIPTION
**Pull request by ZappoMan**
This PR adds support for setting foot IK targets using the controller interface (similar to how hands work).

To test:
- Using a Vive or a Oculus - you can run `script-archive/controllers/puppetFeet2.js` and the left foot will be puppetted by your left hand.
- You can run `script-archive/controllers/proceduralFootPoseExample.js` and watch the left foot be procedurally animated arou
- Using a Kinect you can move your feet and watch them move around when you walk in front of the camera

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 53c439ffba3e4cc105d011b42ed207a4b6fb68c3 available at: https://gist.github.com/548309ea22b213ca98be761b62ddbeff<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10080-53c439f.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10080-53c439f.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 4da8e5e18f3d211637d508422d08dddf09b0d4ea available at: https://gist.github.com/548309ea22b213ca98be761b62ddbeff<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10080-4da8e5e.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10080-4da8e5e.dmg

File: plugins/hifiKinect/src/KinectPlugin.cpp [hunk 1]
```diff
@@ -113,24 +113,23 @@ static controller::StandardPoseChannel KinectJointIndexToPoseIndexMap[KinectJoin
     controller::RIGHT_FORE_ARM,
     controller::RIGHT_HAND,
 
-    controller::RIGHT_UP_LEG,   // hip socket
```
Comment (ZappoMan) [inline plugins/hifiKinect/src/KinectPlugin.cpp]: I had the indices in the wrong order.

**Comment by AndrewMeadows** [comment]
Al and I tested this. QA PASS.

**Pull request merged**